src/components/core/DpUpload/DpUploadFiles.vue

```diff
@@ -88,6 +94,12 @@ export default {
       default: false
     },
 
+    getFileByHash: {
+      type: Function,
+      required: true,
+      default: () => ({})
```

Comment:
I would prefer changing the data-flow a bit and not passing the route, but adding or changing the already uploaded files object with data for the path. (maybe we can use the sessionstorage too? But i am not sure)

Reply:
I don't fully understand what you are suggesting, but I think we should try to get this PR merged soon, instead of making it even bigger. I agree that some things should be solved differently, but maybe this is something to leave for a later PR?

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/91/commits/9c74850fed83b906efbf4365208d9e22f7a5c61b

Reply:
@salisdemos I would try `use the sessionstorage` in other PR when this PR is finished!

Reply:
We can leave it like this for now, but we have to change that during the component-cleanup.